packages/local_auth/lib/local_auth.dart

```diff
@@ -101,6 +101,8 @@ class LocalAuthentication {
     bool biometricOnly = false,
   }) async {
     assert(localizedReason != null);
```

Comment:
Since this plugin has migrated to null safety, this line is actually not necessary anymore. Feel free to remove it while you are here.

Reply:
👍🏼

Reply:
Done